### PR TITLE
Tagging and ACL put notifications

### LINKED ISSUE #361: Add tagging and acl events
* s3:ObjectTagging:*
* s3:ObjectTagging:Put
* s3:ObjectTagging:Delete
* s3:ObjectAcl:Put

### PR DESCRIPTION
**Pull request by masterSplinter01**
Closes #361

### CODE REVIEW

File: api/handler/acl.go [hunk 1]
```diff
@@ -227,12 +228,12 @@ func (h *handler) updateBucketACL(r *http.Request, astChild *ast, bktInfo *data.
...
 	resAst, updated := mergeAst(parentAst, astChild)
 	if !updated {
-		return nil
+		return updated, nil
 	}
 
 	table, err := astToTable(resAst)
 	if err != nil {
-		return fmt.Errorf("could not translate ast to table: %w", err)
+		return updated, fmt.Errorf("could not translate ast to table: %w", err)
```
Comment (KirillovDenis) [inline api/handler/acl.go:236]: I think we should return `false` as `updated` in any error

File: api/handler/acl.go [hunk 2]
```diff
@@ -243,10 +244,10 @@ func (h *handler) updateBucketACL(r *http.Request, astChild *ast, bktInfo *data.
 	}
 
 	if err = h.obj.PutBucketACL(r.Context(), p); err != nil {
-		return fmt.Errorf("could not put bucket acl: %w", err)
+		return updated, fmt.Errorf("could not put bucket acl: %w", err)
```
Comment (KirillovDenis) [inline api/handler/acl.go:247]: I think we should return `false` as `updated` in any error

Comment (KirillovDenis) [inline api/handler/acl.go:247]: I think we should return `false` as `updated` in any error

File: api/handler/acl.go [hunk 3]
```diff
@@ -227,12 +228,12 @@ func (h *handler) updateBucketACL(r *http.Request, astChild *ast, bktInfo *data.
 
 	resAst, updated := mergeAst(parentAst, astChild)
 	if !updated {
-		return nil
+		return updated, nil
```
Comment (KirillovDenis) [inline api/handler/acl.go:231]: Let's return explicitly `false`

Comment (KirillovDenis) [inline api/handler/acl.go:231]: Let's return explicitly `false`

File: api/handler/acl.go [hunk 4]
```diff
@@ -243,10 +244,10 @@ func (h *handler) updateBucketACL(r *http.Request, astChild *ast, bktInfo *data.
 	}
 
 	if err = h.obj.PutBucketACL(r.Context(), p); err != nil {
-		return fmt.Errorf("could not put bucket acl: %w", err)
+		return updated, fmt.Errorf("could not put bucket acl: %w", err)
 	}
 
-	return nil
+	return updated, nil
```
Comment (KirillovDenis) [inline api/handler/acl.go:250]: Let's return explicitly `true`

Comment (KirillovDenis) [inline api/handler/acl.go:250]: Let's return explicitly `true`

**Review by alexvanin — APPROVED** [review]
Consider returning `true` and `false` values explicitly in `updateBucketACL` as @KirillovDenis mentioned.

**Pull request merged**